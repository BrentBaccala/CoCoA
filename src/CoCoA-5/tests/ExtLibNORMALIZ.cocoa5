NmzComputation;

Use QQ[x,y,z];

sorted(NmzNormalToricRing([x^3, x^2*y, y^3]));
sorted(NmzIntClosureToricRing([x^3, x^2*y, y^3]));
sorted(NmzIntClosureMonIdeal([x^3, x^2*y, y^3, x*y^2]));
sorted(NmzIntClosureMonIdeal([x^3, x^2*y, y^3, x*y^2],z));
sorted(NmzEhrhartRing([x^3, x^2*y, y^3, x*y^2],z));
