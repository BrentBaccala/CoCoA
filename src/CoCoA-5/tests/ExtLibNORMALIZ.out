
# <function>

# 
# 
# 
# [y^3, x*y^2, x^2*y, x^3]

# [y, x]

# [y^3, x*y^2, x^2*y, x^3]

# [y, x, y^3*z, x*y^2*z, x^2*y*z, x^3*z]

# [y^3*z, x*y^2*z, x^2*y*z, x^3*z]

# 